specs/openapi/botnim.yaml

```diff
@@ -7,10 +7,28 @@
   },
   "servers": [
     {
-      "url": "https://staging.botnim.co.il"
+     // "url": "https://www.botnim.co.il"
```

Comment:
this doesn't seem right.
this file should be the one matching our staging endpoints.
when deploying to production, we fix the link and remove the `__dev` suffixes.